extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/DevUIProcessor.java

```diff
@@ -63,11 +63,13 @@
 import io.quarkus.maven.dependency.GACTV;
 import io.quarkus.qute.Qute;
 import io.quarkus.runtime.util.ClassPathUtils;
+import io.quarkus.vertx.http.deployment.FilterBuildItem;
```

Comment:
I think this is unused.

Reply:
It's used to get the value of `FilterBuildItem.CORS` (line 141)